ui/src/dashboards/components/Dashboard.js

```diff
@@ -38,11 +44,53 @@ const Dashboard = ({
   })
 
   return (
-    <div className={classnames({'page-contents': true, 'presentation-mode': inPresentationMode})}>
-      {cells.length ?
-        <div className={classnames('container-fluid full-width dashboard', {'dashboard-edit': isEditMode})}>
-          <LayoutRenderer
+    <div
+      className={classnames(
+        'dashboard container-fluid full-width page-contents',
+        {'presentation-mode': inPresentationMode}
+      )}
+    >
+      <div className="template-control-bar">
+        <div className="page-header__left">
+          Template Variables
+        </div>
+        <div className="page-header__right">
+          {templates.map(({id, values}) => {
+            let selected
+            const items = values.map(value => {
+              if (value.selected) {
+                selected = value.value
+              }
+              return {...value, text: value.value}
+            })
+            if (!selected && items.length) {
+              selected = items[0].value
+            }
```

Comment:
Cool, looks good. But how about we refactor to assign `selected` on data creation so that we don't encounter this in the first place?